### DISCUSSION TITLE
Image Size Errors

### DISCUSSION THREAD

**Discussion by TomInVa57** [TumblThreeApp/TumblThree#551] [Q&A (Question & Answer)]
Started getting a lot of "Image size couldn't be retrieved for a post in (xxxxxxx)" error yesterday. Anyone else having this issue? Is it Tumblr or the program. TIA.

**Comment by ErikBrown2** [comment]
Yes, I also see the same error frequently.

**Comment by Someonemustnothavethis** [comment]
Ditto here.  I let it update for my monthly 'catch-up' of my 200+ blogs and immediately started getting these.  Almost every blog gives lots of these.  Clicking them to close is getting tiring.  Tumblr must have changed something again, probably on purpose to interfere with T3 and other Tumblr-scraping sites.
(Next day...)  Watching it most of the day yesterday, this error seems to be keeping it from downloading **some** images.   This morning I'm re-scanning those blogs that didn't completely update (as compared to checking the blogs in a browser) and it grabs a few more, but the same errors continue.

**Reply by TomInVa57** [reply]
Yes, and it seems to be getting worse. I added several blogs to TumblThree, multiple size errors and as you say, only downloads 2 or 3 pics if I'm lucky, some were 0. Tumblr must have made some changes or there having issues, maybe they need a swift kick (reboot) :)

**Comment by caoyi555** [comment]
same problem

**Comment by johnflint1985** [comment]
Same problem

**Comment by thomas694** [comment]
The page structure has changed and TumblThree could no longer find the required data. It's fixed in the new version.

**Reply by Someonemustnothavethis** [reply]
Thanks Thomas!  I wish there was a way I could at least buy you a coffee..or a beer.

**Reply by thomas694** [reply]
<h1>🍻</h1> Cheers!
